docs/reference-manual/common-practices/2-switches.md

```diff
@@ -0,0 +1,95 @@
+
+![End trigger switch](/score-docs/assets/images/reference-manual/common-practices/triggerEnd.gif "End trigger switch")
+
+## On / Off
+
+Combining both the above options, an interval can be toggled on and off at will. This method, like the previous, can also be directly mapped to various devices featuring buttons, keys, hardware switches or any kind of discrete value streams. In this case we will use the first button of a game controller accessed through the [Joystick Device](/score-docs/docs/references/devices-types/joystick-device.html).
+
+![Basic switch](/score-docs/assets/images/reference-manual/common-practices/switch.gif "Basic switch")
+
+When the button is pressed, the first trigger is validated (as by default, it is set to "pulse") and will respond to any value received at the provided address. When the button is released, the end trigger is validated in the same way, before the transition returns to the initial `Sync`.
+
+## Consecutive toggles
+
+The section inside the loop can contain more than one interval. Expanding on the prior exemple, switching between an idol state and an interval, a similar structure could enclose two intervals. This switch also requires an extra trigger. 
+
+![Consecutive switch](/score-docs/assets/images/reference-manual/common-practices/consecutive.gif "Consecutive switch")
+
+With the default "Pulse" operators to validate each trigger, this structure will return to the idol state after each transition. To avoid this extra step and make sure that the first interval executes every time the button's value is true and that the second executes every time it is false, the expression for the trigger has to be modified. By setting both the start and end triggers to only respond when the value is true, they can be validated in a row, skipping the idol state after the transition.  
+
+![Consecutive 2 stages switch](/score-docs/assets/images/reference-manual/common-practices/seq2Stage.gif "Consecutive 2 steps switch")
+
+A better way of achieving the same result would be to rely on a [sub-scenario to isolate the loop](/score-docs/docs/reference-manual/common-practices/1-looping.html#nested-loops). Since the first trigger is only needed to control the very first execution of the loop, it can be validated once to enter the sub-scenario and omitted in the actual toggle structure. The trigger at the end of the sub-scenario is left "never" valid to allow as many repetitions as needed.
+
+![Consecutive switch isolated](/score-docs/assets/images/reference-manual/common-practices/seqIsolate.gif "Consecutive switch isolated")
+
+In context it may be preferable to switch at evry press of the button, instead of having to hold for the first interval and let go for the second. Adding a minimum duration on each of the toggled intervals ensures that the value of the button can go back to false without validating the next trigger. This method also protects the switch from accidental "double taps".
```

Comment:
in context : ? 
evry => every

Reply:
done with 4219439

removed "In context", it's not needed